plugins/entando-plugin-jpkiebpm/src/main/db/update/scriptServ.sql

```diff
@@ -17,3 +17,7 @@ WITH (
 );
 ALTER TABLE public.jpkiebpm_kieformoverride
   OWNER TO agile;
+
```

Comment:
We need to capture this in migration/porting instructions on confluence

Reply:
Done. I had also forgot another column...